### PR TITLE
Update doc for gce_persistent_disk_csi_driver_config

### PR DESCRIPTION
**Pull request by hao-nan-li**
fixes [10035](https://github.com/hashicorp/terraform-provider-google/issues/10035)

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb [hunk 1]
```diff
@@ -380,7 +380,7 @@ func ResourceContainerCluster() *schema.Resource {
 							Computed:     true,
 							AtLeastOneOf: addonsConfigKeys,
 							MaxItems:     1,
-							Description:  `Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Defaults to enabled; set disabled = true to disable.`,
+							Description:  `The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.`,
```
Comment (zli82016) [inline mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb:383]: Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver.

**Note:**: The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.

File: mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 2]
```diff
@@ -426,7 +426,7 @@ Enable/Disable Security Posture API features for the cluster. Structure is [docu
     All cluster nodes running GKE 1.15 and higher are recreated.**
 
 * `gce_persistent_disk_csi_driver_config` - (Optional).
-    Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Defaults to disabled; set `enabled = true` to enabled.
+    The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.
```
Comment (zli82016) [inline mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown:429]:
```
Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver.

**Note:** The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.
```

**Comment by zli82016** [comment]
Should this PR be merged to main branch as it is just doc change?

**Comment by hao-nan-li** [comment]
> Should this PR be merged to main branch as it is just doc change?

This was tagged as a breaking change previously. Hold on a bit I'd need to double-check if there's anything I'm missing.

File: mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb (hunk 1 above)
Comment (hao-nan-li) [reply mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb:383]: Do you mean leaving both descriptions on this field?

Comment (zli82016) [reply mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb:383]: Yeah. And adding the note.

**Comment by c2thorn** [comment]
From https://github.com/GoogleCloudPlatform/magic-modules/pull/8897#discussion_r1321763746, can we update this to just be in `main`?

File: mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb [hunk 3]
```diff
@@ -380,7 +380,7 @@ func ResourceContainerCluster() *schema.Resource {
 							Computed:     true,
 							AtLeastOneOf: addonsConfigKeys,
 							MaxItems:     1,
-							Description:  `Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Defaults to enabled; set disabled = true to disable.`,
+							Description:  `Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Set enabled = true to enabled. The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.`,
```
Comment (c2thorn) [inline mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb:383]:
```suggestion
							Description:  `Whether this cluster should enable the Google Compute Engine Persistent Disk Container Storage Interface (CSI) Driver. Set enabled = true to enable. The Compute Engine persistent disk CSI Driver is enabled by default on newly created clusters for the following versions: Linux clusters: GKE version 1.18.10-gke.2100 or later, or 1.19.3-gke.2100 or later.`,
```

**Comment by hao-nan-li** [comment]
> From [#8897 (comment)](https://github.com/GoogleCloudPlatform/magic-modules/pull/8897#discussion_r1321763746), can we update this to just be in `main`?
> 
> Also, no changelog is needed.

Sounds good, I'll rebase it and target the main branch.

**Pull request closed without merging**